cl/opinion_page/templates/view_opinion_authorities.html

```diff
@@ -30,24 +30,23 @@ <h2>
         </h2>
 
         <div id="authorities">
-            <h3>This opinion cites {{ authorities.count|intcomma }} opinion{{ authorities.count|pluralize }}.</h3>
+            <h3>This opinion cites {{ authorities_with_data|length|intcomma }} opinion{{ authorities_with_data|length|pluralize }}.</h3>
             <hr>
             {# Westlaw's metadata organization: Checkbox, Treatment, Name + citation + summary, Date, Type, Depth #}
             <ul>
-                {% for authority in authorities %}
+                {% for authority in authorities_with_data %}
                     <li>
+                        {{ authority.citation_depth }} reference{{ authority.citation_depth|pluralize }} to
                         <a href="{{ authority.get_absolute_url }}?{{ request.META.QUERY_STRING }}">
                             {{ authority.caption|safe|v_wrapper }}
                         </a>
                         <br/>
                         {{ authority.docket.court }} |
                         {{ authority.date_filed }} |
                         {% if authority.citation_count > 0 %}
```

Comment:
Should this be `> 1`, since we're doing the `-1` stuff?

Reply:
Fixed in 4d842b2